plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/StaticMetastoreConfig.java

```diff
@@ -21,18 +21,20 @@
 
 import java.net.URI;
 import java.util.List;
+import java.util.Optional;
 
 import static com.google.common.collect.ImmutableList.toImmutableList;
 import static com.google.common.collect.Streams.stream;
 
 public class StaticMetastoreConfig
 {
     public static final String HIVE_METASTORE_USERNAME = "hive.metastore.username";
-
+    public static final String HIVE_METASTORE_CATALOG_NAME = "hive.metastore.catalog-name";
```

Comment:
we don't need to define a constant here - pls use `@Config("hive.metastore.catalog-name")`.

`HIVE_METASTORE_USERNAME` is used as a constant because it is being referenced externally.